kip-0038.md

```diff
@@ -26,81 +37,95 @@ The `kadena_getAccounts_v2` method improved on the previous by allowing extra fe
 
 ### Request Parameters
 
-```typescript
-interface GetAccountsRequest {
-  accounts?: Array<{
-    publicKey: string;
-    contract: string;
```

Comment:
The same thing like above. Also optional. We could add this one later if we want to